mteb/evaluation/evaluators/ClassificationEvaluator.py

```diff
@@ -312,11 +254,16 @@ def __init__(
         y_test,
         task_name: str,
         max_iter: int = 100,
-        batch_size: int = 32,
+        encode_kwargs: dict[str, Any] = {},
         limit: int | None = None,
         **kwargs,
     ):
         super().__init__(**kwargs)
+        self.encode_kwargs = encode_kwargs
+
+        if "batch_size" not in self.encode_kwargs:
+            self.encode_kwargs["batch_size"] = 32
+
```

Comment:
Why not default it to  `encode_kwargs: dict[str, Any] = {"batch_size": 32},` like in the other classes above?

Reply:
Since most places use a default of {} it would just be overwritten. This way we ensure the same behaviour. (I will fix the changed where the encode_kwargs has a default other than {}.